include/deal2lkit/parsed_quadrature.h

```diff
@@ -0,0 +1,90 @@
+//-----------------------------------------------------------
+//
+//    Copyright (C) 2015 by the deal2lkit authors
+//
+//    This file is part of the deal2lkit library.
+//
+//    The deal2lkit library is free software; you can use it, redistribute
+//    it, and/or modify it under the terms of the GNU Lesser General
+//    Public License as published by the Free Software Foundation; either
+//    version 2.1 of the License, or (at your option) any later version.
+//    The full text of the license can be found in the file LICENSE at
+//    the top level of the deal2lkit distribution.
+//
+//-----------------------------------------------------------
+
+#ifndef _d2k_parsed_quadrature
+#define _d2k_parsed_quadrature
+
+#include <deal.II/base/quadrature.h>
+#include <deal2lkit/parameter_acceptor.h>
+
+using namespace dealii;
+
+
+D2K_NAMESPACE_OPEN
+
+/**
+ * A deal2lkit wrapper for dealii::QuadratureSelector.
+ * The template integers specify the dimension of the quadrature.
+ */
+template<int dim>
+class ParsedQuadrature : public Quadrature<dim>, public ParameterAcceptor
+{
+public:
+  /**
+   * Constructor: takes an optional name for the section.
+   * Moreover, it Takes the name of the quadrature rule @p quadrature_type that
+   * could be chosen among following names:
+   * -  gauss;
+   * -  midpoint;
+   * -  milne;
+   * -  simpson;
+   * -  trapez;
+   * -  weddle;
+   * If @p quadrature_type is "gauss" it is possible to specify the number of
+   * quadrature points in each coordinate direction (@p order).
+   */
+  ParsedQuadrature( const std::string   &name="",
+                    const std::string   &quadrature_type="gauss",
+                    const unsigned int  repetitions = 2,
```

Comment:
need to have = throw an exception is it's not zero?

invert **the order of the arguments to the function**, not the values of the default...